desktop-widgets/mainwindow.cpp

```diff
@@ -200,6 +200,7 @@ MainWindow::MainWindow() : QMainWindow(),
 	connect(PreferencesDialog::instance(), SIGNAL(settingsChanged()), divePlannerWidget(), SLOT(settingsChanged()));
 	connect(PreferencesDialog::instance(), SIGNAL(settingsChanged()), divePlannerSettingsWidget(), SLOT(settingsChanged()));
 	connect(PreferencesDialog::instance(), SIGNAL(settingsChanged()), TankInfoModel::instance(), SLOT(update()));
+	// TODO Make number of recent files dynamic (at compile time)
```

Comment:
is this a left over comment?

Reply:
No, this is just a reminder that if NUM_RECENT_FILES is changed, so must be this code. I made this more clear.